src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age !== undefined && (
+      <>
+        <p className="Person__age">{`I am ${person.age}`}</p>
+      </>
+    )}
+    {person.isMarried === false ? (
```

Comment:
```suggestion
    {person.isMarried === false ? (
```

You can use these shorthand

If it is false then use `!person.isMarried`

It it is true then use `person.isMarried`